test/test_suite.py

```diff
@@ -1339,3 +1323,67 @@ def test_enotation_decimal(self):
             [decimal.Decimal("696E-9")],
             [decimal.Decimal("696E-9")],
         )
+
+
+class LikeFunctionsTest(_LikeFunctionsTest):
+    @pytest.mark.skip("Spanner does not support ESCAPE")
+    def test_contains_autoescape(self):
+        pass
+
+    @pytest.mark.skip("Spanner does not support ESCAPE")
+    def test_contains_autoescape_escape(self):
+        pass
+
+    @pytest.mark.skip("Spanner does not support ESCAPE")
+    def test_contains_escape(self):
+        pass
+
+    @pytest.mark.skip("Spanner does not support ESCAPE")
+    def test_endswith_autoescape(self):
+        pass
+
+    @pytest.mark.skip("Spanner does not support ESCAPE")
+    def test_endswith_escape(self):
+        pass
+
+    @pytest.mark.skip("Spanner does not support ESCAPE")
+    def test_endswith_autoescape_escape(self):
+        pass
+
+    @pytest.mark.skip("Spanner does not support ESCAPE")
+    def test_startswith_autoescape(self):
+        pass
+
+    @pytest.mark.skip("Spanner does not support ESCAPE")
+    def test_startswith_escape(self):
+        pass
+
+    @pytest.mark.skip("Spanner does not support ESCAPE")
+    def test_startswith_autoescape_escape(self):
+        pass
+
+
+@pytest.mark.skip("Spanner doesn't support quotes in table names.")
+class QuotedNameArgumentTest(_QuotedNameArgumentTest):
+    pass
+
+
+@pytest.mark.skip("Spanner doesn't support IS DISTINCT FROM clause")
+class IsOrIsNotDistinctFromTest(_IsOrIsNotDistinctFromTest):
+    pass
+
+
+class OrderByLabelTest(_OrderByLabelTest):
+    @pytest.mark.skip("Spanner doesn't supports composed GROUP BY.")
```

Comment:
This skip reason is not quite correct.

For the given query:
```
SELECT count(some_table.id) AS count_1, some_table.x + some_table.y AS lx 
FROM some_table GROUP BY some_table.x + some_table.y ORDER BY lx
```
Spanner isn't recognizing that the `some_table.x + some_table.y` in the `GROUP BY` list is the same as the `some_table.x + some_table.y` in the `SELECT` list.

The explanation I was given:
> You can use the derived expression in the GROUP BY, but if you want to use it in the SELECT list too, you have to use an alias or some other sort of label to clue Spanner in on the fact that they're the same thing.

So the valid version of this query would be:
```
SELECT count(some_table.id) AS count_1, some_table.x + some_table.y AS lx 
FROM some_table GROUP BY lx ORDER BY lx
```

I think this may be a more accurate message:
`Spanner requires an alias for the GROUP BY list when specifying derived columns also used in SELECT`